src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -0,0 +1,488 @@
+                      name="venue.auditoriumColumns"
+                      type="number"
+                      ref={register}
+                      className="align-left"
+                      placeholder="Number of seats columns"
+                    />
+                    {errors?.venue?.auditoriumColumns ? (
+                      <span className="input-error">
+                        {errors?.venue?.auditoriumColumns.message}
+                      </span>
+                    ) : null}
+                  </div>
+                  <div className="input-container">
+                    <h4 className="italic input-header">
+                      Number of seats rows
+                    </h4>
+                    <input
+                      defaultValue={DEFAULT_AUDIENCE_ROWS_NUMBER}
+                      name="venue.auditoriumRows"
+                      type="number"
+                      ref={register}
+                      className="align-left"
+                      placeholder="Number of seats rows"
+                      min={5}
+                    />
+                    {errors?.venue?.auditoriumRows ? (
+                      <span className="input-error">
+                        {errors?.venue?.auditoriumRows.message}
+                      </span>
+                    ) : null}
+                  </div>
+                </>
+              )}
+
+              {room.template &&
+                HAS_GRID_TEMPLATES.includes(room.template as VenueTemplate) &&
+                venueValues.showGrid && (
+                  <>
+                    <div className="input-container">
+                      <h4 className="italic input-header">Number of columns</h4>
+                      <input
+                        defaultValue={1}
+                        name="venue.columns"
+                        type="number"
+                        ref={register}
+                        className="align-left"
+                        placeholder={`Number of grid columns`}
+                      />
+                      {errors?.venue?.columns ? (
+                        <span className="input-error">
+                          {errors?.venue?.columns.message}
+                        </span>
+                      ) : null}
+                    </div>
+                    <div className="input-container">
+                      <h4 className="italic input-header">Number of rows</h4>
+                      <div>
+                        Not editable. The number of rows is derived from the
+                        number of specified columns and the width:height ratio
+                        of the party map, to keep the two aligned.
+                      </div>
+                    </div>
+                  </>
+                )}
+            </>
+          )}
+
+          <Button
+            disabled={isUpdating || isDeleting}
+            onClick={deleteSelectedRoom}
+          >
+            Delete room
+          </Button>
```

Comment:
This one button can safely be updated. 
```suggestion
          <ButtonNG
            disabled={isUpdating || isDeleting}
            loading={isUpdating || isDeleting}
            onClick={deleteSelectedRoom}
            variant="danger"
          >
            Delete room
          </ButtonNG>
```



[OOS]
As for the bottom ones they should probably become part of an improved `TabFooter` one that allows for different types/numbers of buttons to accommodate design. Probably new PR under https://github.com/sparkletown/internal-sparkle-issues/issues/1195

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2312/commits/0023b3b4d849d0af4ae51d5af3ca558495cb927e